docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+
+4. To fix the issue, create a new branch (for example, `git checkout -b fix-the-thing`) from the `support/<CURRENT MAJOR VERSION NUMBER>.x` branch.
+
+5. Run `npm install` to make sure you have the latest dependencies installed.
+
+6. Make your code changes, and test them following our [standard testing requirements](/docs/contributing/testing.md).
+
+7. Update the changelog with details of the fix.
+
+8. Commit your changes.
+
+9. Push your new branch (see step 4) to GitHub and raise a pull request, with `support/<CURRENT MAJOR VERSION NUMBER>.x` as the base branch to merge into.
+
+10. Once the PR is approved, merge it into `support/<CURRENT MAJOR VERSION NUMBER>.x`. It’s usually a developer who reviews the PR, but sometimes PRs need an extra review from another role.
+
+#### Build a new release
+
+1. Check out `support/<CURRENT MAJOR VERSION NUMBER>.x`.
+
+2. Create and check out a new branch, `support-release-[version-number]`. The version number of the new release depends on the type of release. New features correspond to a minor (X.1.X) change - for example, '3.14.0 (Feature release)'. Fixes correspond to a patch (X.X.1) change - for example, '3.13.1 (Patch release)'. In either case, refer to the previous release of that kind, and give the new release the logical next number. To learn more about our versioning, see our [guidance on updating the changelog](/docs/contributing/versioning.md#updating-changelog).
+
+3. Run `nvm use` to make sure you’re using the right version of Node.js and npm.
+
+4. Run `npm install` to make sure you have the latest dependencies installed
+
+5. In the CHANGELOG.md, replace the 'Unreleased' heading with the new version number and its release type. For example, '3.14.1 (Fix release)'. Also add a new 'Unreleased' heading above this new heading, so people raising new PRs know where to add them in the changelog.
+
+6. Update the `package/package.json` version with the new version number.
+
+7. Save the changes. Do not commit.
+
+8. Run `npm run build-release`, which will prompt you to either continue or cancel. Enter `y` to continue.
+
+9. If you want to make sure your changes work when used in the GOV.UK Design System, use [`npm-link`](https://docs.npmjs.com/cli/v7/commands/npm-link) to test before publishing:
+
+  ```bash
+  cd ../govuk-design-system
+  git checkout main
+  git pull
+  npm install # note running `npm install` after `npm link` will destroy the link.
+  npm link ../govuk-frontend/package/
+  ```
+
+10. When you finish testing, run `npm unlink ../govuk-frontend/package/` to unlink the package.
+
+11. Raise a pull request, with `support/<CURRENT MAJOR VERSION NUMBER>.x` as the base branch to merge into.
+
+12. Once the pull request is approved, merge it into `support/<CURRENT MAJOR VERSION NUMBER>.x`.
+
+#### Publish the release to npm
+
+1. Check out `support/<CURRENT MAJOR VERSION NUMBER>.x` and pull the latest changes.
+
+2. Run `npm login` to log into npm using our team credentials.
```

Comment:
Sign in rather than log in, as per style guide

Reply:
Ah, didn't know about this - thanks, will keep in mind!